.golangci.yml

```diff
@@ -1,24 +1,18 @@
-run:
-  timeout: 5m
-
+version: "2"
```

Comment:
Nice, was this automatically converted? I see a couple oif linters are gone (e.g. `gosimple` and `typecheck`)

Reply:
Yes, this was automatically converted using `golangci-lint migrate`.  The removed linters are gone from upstream anyway.

Reply:
From the migration guide:

https://golangci-lint.run/product/migration-guide/#lintersenableformatter_name